### PR TITLE
Rewrite OS backend sections

### PR DESCRIPTION
**Pull request by karlowich**

### CODE REVIEW

**Review by safl — DISMISSED** [review]
Great!

File: docs/backends/linux/index.rst [hunk 1]
```diff
@@ -3,42 +3,55 @@
...
-...
+.. _sec-backends-linux-configuration:
 
-Device Identifiers
-------------------
+System Configuration
+--------------------
+
+To use the Linux backend you need to be running a Linux distribution. We
+support a number of Linux distributions, we recommend using one of the
```
Comment (NaddiNadja) [inline docs/backends/linux/index.rst:12]: without a conjunction after the comma, it sounds a bit weird ... for macOS and freebsd, you write "..., however to ensure capability ..."

**Review by NaddiNadja — CHANGES_REQUESTED** [review]

File: docs/backends/linux/index.rst (hunk 1 above)
Comment (karlowich) [reply docs/backends/linux/index.rst]: I changed the wording, it should be better now

**Review by NaddiNadja — APPROVED** [review]
lgtm :)

File: .github/workflows/verify.yml [hunk 2]
```diff
@@ -662,7 +662,8 @@ jobs:
     - name: Runnner-prep, clean up self-hosted left-overs
       run: |
         pkill -f qemu || true
-        rm -r $HOME/guests || true
+        rm -rf $HOME/guests
```
Comment (safl) [inline .github/workflows/verify.yml:665]: Great!

**Review by safl — APPROVED** [review]
Excellent!

**Pull request merged**